example_python/main.py

```diff
@@ -0,0 +1,110 @@
+
+# HTTP API for fetching available data streams
+async def get_datastreams(request: web.Request) -> web.Response:
+    response = {
+        "datastreams": [{"UUID": ds["UUID"], "color": ds["color"]} for ds in DATASTREAMS]
+    }
+    return web.json_response(response)
+
+# WebSocket connection handling
+async def websocket_handler(request):
+    ws = web.WebSocketResponse()
+    await ws.prepare(request)
+    print("Client connected to WebSocket")
+    try:
+        async for message in ws:
+            command = message.data.strip().split()
+            if not command:
+                continue  # Ignore empty commands
+
+            uuids = command[:-2] if len(command) > 2 else command
+            uuids = [uuid for uuid in uuids if uuid in {ds["UUID"] for ds in DATASTREAMS}]
+            if not uuids:
+                await ws.send_json({"error": "No valid UUIDs provided"})
+                continue
+
+            sample_rate = int(command[-2]) if len(command) > 2 and command[-2].isdigit() else 60
+            output_format = command[-1] if len(command) > 2 else "json"
+
+            selected_datastreams = [ds for ds in DATASTREAMS if ds["UUID"] in uuids]
+            device_order = [ds["UUID"] for ds in selected_datastreams]
+
+            async def send_data():
+                while not ws.closed:
+                    real_timestamp = round(time.time(), 3)  # UNIX-Zeit mit Millisekunden
+                    # Alternative: ISO 8601 Zeitformat
+                    # real_timestamp = datetime.datetime.now().isoformat()
+
+                    sample_data = {
+                        "timestamp": real_timestamp,
+                        "datastreams": device_order,
+                        "data": [[
+                            round(generate_sine_wave(real_timestamp), 3) if ds["type"] == "sine" else generate_square_wave(real_timestamp)
+                            for ds in selected_datastreams
+                        ]]
+                    }
+                    
+                    if output_format == "json":
+                        await ws.send_json(sample_data)
+                    elif output_format == "csv":
+                        csv_data = f"{real_timestamp}," + ",".join(map(str, sample_data["data"][0]))
+                        await ws.send_str(csv_data)
+
+                    await asyncio.sleep(1 / sample_rate)
+            
+            task = asyncio.create_task(send_data())
+            await task
+    except Exception as e:
+        print(f"WebSocket error: {e}")
+    finally:
+        print("Client disconnected")
+    return ws
+
+# Create aiohttp application
+app = web.Application()
+app.router.add_get("/UUID", get_datastreams)
+app.router.add_get("/ws", websocket_handler)
+
+# Start the server
+async def main() -> None:
+    runner = web.AppRunner(app)
+    await runner.setup()
+    site = web.TCPSite(runner, "0.0.0.0", 8080)
```

Comment:
To make it easy to set the port, we might want to create a variable with that name to help people understand the code
```suggestion
    port = 8080
    site = web.TCPSite(runner, "0.0.0.0", port )
```